x-pack/legacy/plugins/apm/public/components/app/TransactionOverview/List/index.tsx

```diff
@@ -104,7 +103,7 @@ export function TransactionList({ items, serviceName, isLoading }: Props) {
         render: (value: number) => <ImpactBar value={value} />
       }
     ],
-    [serviceName]
+    []
```

Comment:
Did the lint rule catch this as well (that it's not needed)?

Reply:
yep!